modules/administration-guide/partials/snip_configuring-network-policies.adoc

```diff
@@ -0,0 +1 @@
+* link:https://docs.openshift.com/container-platform/{ocp4-ver}/networking/network_policy/multitenant-network-policy.html[Configuring multitenant isolation with network policy]
```

Comment:
Unless you have any specific reason to keep this as a separate snip file, I suggest deleting this file.

Reply:
I'm good with deleting it